### PR TITLE
updated the before bloc to use API call

### PR DESCRIPTION
**Pull request by bwelch8105**
I also made other changes to correct current issues with tests

### CODE REVIEW

File: cypress/integration/WebInterface/Measure/EditMeasure/EditMeasureAddMetaData.spec.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ describe('Edit Measure: Add Meta Data', () => {
         OktaLogin.Login()
```
Comment (cvasile001) [inline cypress/integration/WebInterface/Measure/EditMeasure/EditMeasureAddMetaData.spec.ts:16]: we should not need log in and log out, these lines c an be deleted if we are seeding data with the api

File: cypress/integration/WebInterface/Measure/EditMeasure/EditMeasureAddMetaData.spec.ts [hunk 2]
```diff
@@ -66,7 +66,7 @@ describe('Edit Measure: Add Meta Data', () => {
         cy.get(EditMeasurePage.measureDisclaimerSaveButton).click()
         cy.get(EditMeasurePage.measureDisclaimerSuccessMessage).should('be.visible')
 
-        cy.get(Header.measures).click()
```
Comment (cvasile001) [inline cypress/integration/WebInterface/Measure/EditMeasure/EditMeasureAddMetaData.spec.ts:69]: lets try to use the header here, if we are clicking on the header

File: cypress/integration/WebInterface/Measure/EditMeasure/EditMeasureNameValidations.spec.ts [hunk 3]
```diff
@@ -2,8 +2,22 @@ import {OktaLogin} from "../../../../Shared/OktaLogin"
...
 import {EditMeasurePage} from "../../../../Shared/EditMeasurePage"
 import {MeasuresPage} from "../../../../Shared/MeasuresPage"
+let measureName = 'TestMeasure' + Date.now()
+let CqlLibraryName = 'TestLibrary' + Date.now()
+let measureScoring = 'Ratio'
 
 describe('Edit Measure Name Validations', () => {
+    before('Create Measure', () => {
+
+        OktaLogin.Login()
```
Comment (cvasile001) [inline cypress/integration/WebInterface/Measure/EditMeasure/EditMeasureNameValidations.spec.ts:12]: another spot where we can remove the login and logout

File: cypress/integration/WebInterface/Smoke Tests/EditMeasure.spec.ts [hunk 4]
```diff
@@ -4,12 +4,23 @@ import {EditMeasurePage} from "../../../Shared/EditMeasurePage"
 import {MeasuresPage} from "../../../Shared/MeasuresPage"
 import {Header} from "../../../Shared/Header"
 
-let measureName = 'TestMeasure' + Date.now()
-let CqlLibraryName = 'TestLibrary' + Date.now()
+let measureName = 'TestMeasure' + Date.now() + 1
+let CqlLibraryName = 'TestLibrary' + Date.now() + 1
```
Comment (cvasile001) [inline cypress/integration/WebInterface/Smoke Tests/EditMeasure.spec.ts:8]: not sure what the +1 is, but cql Library has strict naming rules, I would double check that those rules are passing with this name

File: cypress/integration/WebInterface/Smoke Tests/EditMeasure.spec.ts [hunk 5]
```diff
@@ -40,7 +51,9 @@ describe('Edit Measure', () => {
         cy.get(EditMeasurePage.measureStewardConfirmaionText).should('contain.text', 'Measure Steward Information Saved Successfully')
 
         //Navigate back to Measures page and verify if the Measure Name is updated
-        cy.get(Header.measures).click()
```
Comment (cvasile001) [inline cypress/integration/WebInterface/Smoke Tests/EditMeasure.spec.ts:43]: why did we need to make this change? is clicking measures in the header not working?

File: cypress/Shared/TestCasesPage.ts [hunk 6]
```diff
@@ -9,6 +9,8 @@ export class TestCasesPage {
     public static readonly createTestCaseButton = '[data-testid=create-test-case-button]'
     public static readonly successMsg = '[data-testid="create-test-case-alert"]'
     public static readonly testCaseTitleList = 'tbody > tr > :nth-child(2)'
+    //#main > div > div > div > div.TestCaseLanding___StyledDiv-sc-1mkj7s4-0.keFWdW > section:nth-child(2) > div > div > div.TestCaseList___StyledDiv3-sc-1iefzo5-4.fSPsSe > div > table > tbody > tr:nth-child(1) > td:nth-child(2)
+    //<td>Test Case</td>
```
[the remark this answers is not in the document]
Comment (bwelch8105) [reply cypress/Shared/TestCasesPage.ts]: comment has been removed

File: cypress/Shared/TestCasesPage.ts [hunk 7]
```diff
@@ -36,6 +38,12 @@ export class TestCasesPage {
         })
     }
 
+    public static grabValidateTestCaseTitle(testCaseTitle: string) : void{
+        cy.readFile('cypress/downloads/testCaseId').should('exist').then((fileContents) => {
+            cy.get('[data-testid=test-case-row-'+ fileContents +']').should('be.visible').click().contains(testCaseTitle)
```
[the remark this answers is not in the document]
Comment (bwelch8105) [reply cypress/Shared/TestCasesPage.ts]: click action has been removed

File: cypress/integration/WebInterface/Measure/EditMeasure/EditMeasureAddMetaData.spec.ts (hunk 1 above)
Comment (bwelch8105) [reply cypress/integration/WebInterface/Measure/EditMeasure/EditMeasureAddMetaData.spec.ts]: unnecessary logins and logouts have been removed

File: cypress/integration/WebInterface/Measure/EditMeasure/EditMeasureNameValidations.spec.ts (hunk 3 above)
Comment (bwelch8105) [reply cypress/integration/WebInterface/Measure/EditMeasure/EditMeasureNameValidations.spec.ts]: unnecessary logins and logouts have been removed

File: cypress/integration/WebInterface/Measure/CreateMeasure/CreateDifferentMeasureTypes.spec.ts [hunk 8]
```diff
@@ -21,7 +21,7 @@ describe('Create different Measure types', () => {
         CqlLibraryName = 'CohortTestLibrary' + Date.now()
         measureScoring = 'Cohort'
```
[the remark this answers is not in the document]
Comment (bwelch8105) [reply cypress/integration/WebInterface/Measure/CreateMeasure/CreateDifferentMeasureTypes.spec.ts:23]: tests that need a measure created via uI have been reverted to use the UI

**Review by cvasile001 — APPROVED** [review]
Looks good!

**Pull request merged**